eZ/Publish/Core/FieldType/RichText/Type.php

```diff
@@ -266,6 +274,13 @@ public function validate(FieldDefinition $fieldDefinition, SPIValue $value)
             );
         }
 
+        if ($this->internalLinkValidator !== null) {
+            $errors = $this->internalLinkValidator->validateDocument($value->xml);
+            foreach ($errors as $error) {
+                $validationErrors[] = new ValidationError($error);
+            }
+        }
```

Comment:
I considered making this a part of `\eZ\Publish\Core\FieldType\RichText\Validator`, but on secod thought it's a good responsibility separation.
+1